packages/office-ui-fabric-react/src/components/TextField/TextField.doc.tsx

```diff
@@ -6,12 +6,10 @@ import { TextFieldBorderlessExample } from './examples/TextField.Borderless.Exam
 import { TextFieldCustomRenderExample } from './examples/TextField.CustomRender.Example';
 import { TextFieldErrorMessageExample } from './examples/TextField.ErrorMessage.Example';
 import { TextFieldMultilineExample } from './examples/TextField.Multiline.Example';
-import { TextFieldPlaceholderExample } from './examples/TextField.Placeholder.Example';
```

Comment:
We were using the placeholder examples when verifying the fluent styles for placeholder text. Can we include them back maybe within the basic examples?

Reply:
I combined the placeholder examples and basic examples.